lib/project_types/extension/models/specification_handlers/default.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Extension
+  module Models
+    module SpecificationHandlers
+      class Default
```

Comment:
`Extension::Models::Type -> Extension::Models::SpecificationHandlers` – only the name and level of nesting has change. The file contents itself hasn't been altered further.